configurations/arm/ubuntu2204.yaml

```diff
@@ -0,0 +1,8 @@
+ami_db_scylla_user: 'ubuntu'
+ami_id_db_scylla: 'ami-022c8ce295ce9ac4c'  # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-arm64-server-20220506
+region_name: 'eu-west-1'
+instance_type_db: 'im4gn.xlarge'
```

Comment:
@KnifeyMoloko please send another PR to change that also in all other arm artifact tests.

Reply:
Sure thing: https://github.com/scylladb/scylla-cluster-tests/pull/4811